server/routes.js

```diff
@@ -1,19 +1,67 @@
 import { Router } from 'express';
 import Entry from './entry';
-import EntryController from './controllers/EntryController';
 
 const appRouter = Router();
 
-appRouter.get('/', (req, res) => res.send('Welcome to MyDiary API End Points!'));
+const entries = [
+  new Entry(1, 'My moms 50th birthday', 'Oh! What a lovely day it was to see her so happy'),
+  new Entry(2, 'My birthday', 'Happy birthday to meeeee'),
+  new Entry(3, 'My last day at school', 'I have waited for this day all my life'),
+  new Entry(4, 'My first bicycle', 'Your fave does not have a bike as cool as mine'),
+];
 
-appRouter.get('/api/v1/entries', EntryController.getEntries);
+appRouter.get('/', (request, response) => response.send('Welcome to MyDiary API End Points!'));
```

Comment:
Could you revert your changes on this?
Separate your routes from your controllers.